interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -562,7 +562,7 @@ public void exchangeMetadataUnaryCall() throws Exception {
     Assert.assertEquals(contextValue, trailersCapture.get().get(METADATA_KEY));
   }
 
-  @Test(timeout = 10000)
+  @Test(timeout = 1000000000)
```

Comment:
Did you revert the wrong test timeout? (Not to say that it shouldn't also use a lower timeout)